### PR TITLE
Handle absolute registry paths

### PR DESCRIPTION
**Pull request by kyork-cl**
I added a check for absolute pathing so that it would use the path provided instead of appending to cwd()

According to the docs, `path.isAbsolute()` is only supported in node v0.12+.  Is that compatible with your requirements?

### CODE REVIEW

File: lib/registry.js [hunk 1]
```diff
@@ -14,7 +14,11 @@ var Registry = module.exports = function(opts){
   if (opts.db) {
     this.db = opts.db;
   } else {
-    this.location = path.join(process.cwd(), opts.path);
```
Comment (mdobson) [inline lib/registry.js]: If we changed this to be easier to override work for you @kyork-cl ? I'd prefer not to break backwards compatibility with node.

Something like this?

```
this.location = opts.path;
```

Comment (kyork-cl) [inline lib/registry.js]: Sure

Comment (mdobson) [inline lib/registry.js]: Cool. I'll amend the PR.

Comment (kyork-cl) [inline lib/registry.js]: Too slow!

**Pull request merged**